week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, age) {
```

Comment:
Great!  This is correct.  If I wanted to be REALLY unfair I could say that your greeting will be grammatically incorrect if the person's age is `1`.